src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -503,6 +503,10 @@ SELECT filter(count(*), WHERE result = 'FAILED') FROM SyntheticCheck WHERE monit
 
 In this example, a window with a successful result would return a `0`, allowing the condition's threshold to resolve on its own.
 
+<Callout variant="tip">
```

Comment:
See above comment about the lack of detail related to WHERE-->increased $$$

Reply:
@homelessbirds can you verify that this is true and/or is language that we should be using here?